barretenberg/cpp/src/barretenberg/vm2/simulation/field_gt.cpp

```diff
@@ -0,0 +1,49 @@
+#include "barretenberg/vm2/simulation/field_gt.hpp"
+
+#include "barretenberg/vm2/common/field.hpp"
+
+namespace bb::avm2::simulation {
+
+bool FieldGreaterThan::ff_gt(const FF& a, const FF& b)
```

Comment:
This whole method feels so so bad from a simulation PoV... but since we have to call the range check gadget I'm not sure there's a far better way...